### ISSUE TITLE
Weird output when using single template.yaml with multiple functions

### ISSUE THREAD

**Issue by andres-lowrie** [awslabs/aws-sam-cli#298]
Hello

So I'm getting some weird output when using a single `template.yaml` file that has multiple functions (...doing a crud style)

**template.yaml**

```
---
Parameters:
  CodeBucket:
    Type: String
    Default: s3://some-bucket.zip
    Description: The bucket to upload the code to. Dont change this w/o a good reason
  Table:
    Type: String
    Default: WebForms
    Description: The dynamo table to use. Dont change this w/o a good reason

Resources:
  WebformsGetFunction:
    Type: AWS::Serverless::Function
    Properties:
      FunctionName: GetWebform
      Handler: webforms/index.get
      Runtime: nodejs6.10
      CodeUri:
        Ref: CodeBucket
      Policies: AmazonDynamoDBReadFullAccess
      Environment:
        Variables:
          TABLE_NAME: Table
      Events:
        GetResource:
          Type: Api
          Properties:
            Path: "/webforms/{formId}"
            Method: get

  WebformsPostFunction:
    Type: AWS::Serverless::Function
    Properties:
      FunctionName: PostWebform
      Handler: webforms/index.post
      Runtime: nodejs6.10
      CodeUri:
        Ref: CodeBucket
      Policies: AmazonDynamoDBFullAccess
      Environment:
        Variables:
          TABLE_NAME: Table
      Events:
        PostResource:
          Type: Api
          Properties:
            Path: "/webforms"
            Method: post

  WebformsPutFunction:
    Type: AWS::Serverless::Function
    Properties:
      FunctionName: PutWebform
      Handler: webforms/index.put
      Runtime: nodejs6.10
      CodeUri:
        Ref: CodeBucket
      Policies: AmazonDynamoDBFullAccess
      Environment:
        Variables:
          TABLE_NAME: Table
      Events:
        PutResource:
          Type: Api
          Properties:
            Path: "/webforms/{formId}"
            Method: put

  WebformsDeleteFunction:
    Type: AWS::Serverless::Function
    Properties:
      FunctionName: DeleteWebform
      Handler: webforms/index.delete
      Runtime: nodejs6.10
      CodeUri:
        Ref: CodeBucket
      Policies: AmazonDynamoDBFullAccess
      Environment:
        Variables:
          TABLE_NAME: Table
      Events:
        DeleteResource:
          Type: Api
          Properties:
            Path: "/webforms/{formId}"
            Method: delete
```

Produces the following output, in particular the lines that mention the "Mounting"

```
...

Mounting webforms/index.delete (nodejs6.10) at http://0.0.0.0:3000/webforms/{formId} [GET]
Mounting webforms/index.delete (nodejs6.10) at http://0.0.0.0:3000/webforms [POST]
Mounting webforms/index.delete (nodejs6.10) at http://0.0.0.0:3000/webforms/{formId} [PUT]
Mounting webforms/index.delete (nodejs6.10) at http://0.0.0.0:3000/webforms/{formId} [DELETE]

...
```

Everything works when making the actual http calls though in terms of the routing, it's just the output that says it's pointing to another function.

Don't know if it's a bug on my end or what.

**Comment by tebruno99** [comment]
I am hitting this today as well. Spent a bit of time trying to modify my template cause I thought it was wrong.

**Comment by jfuss** [comment]
Running `sam local start-api` in the `tests/integration/testdata/start-api` folder. I now see the output

```
2018-05-09 18:42:33 Mounting HelloWorldFunction at http://127.0.0.1:3000/anyandall [GET, DELETE, PUT, POST, HEAD, OPTIONS, PATCH]
2018-05-09 18:42:33 Mounting WriteToStderrFunction at http://127.0.0.1:3000/writetostderr [GET]
2018-05-09 18:42:33 Mounting Base64ResponseFunction at http://127.0.0.1:3000/base64response [GET]
2018-05-09 18:42:33 Mounting OnlySetStatusCodeFunction at http://127.0.0.1:3000/onlysetstatuscode [GET]
2018-05-09 18:42:33 Mounting EchoBase64EventBodyFunction at http://127.0.0.1:3000/echobase64eventbody [POST]
2018-05-09 18:42:33 Mounting HelloWorldFunction at http://127.0.0.1:3000/proxypath/{proxy+} [GET]
2018-05-09 18:42:33 Mounting OnlySetBodyFunction at http://127.0.0.1:3000/onlysetbody [GET]
2018-05-09 18:42:33 Mounting HelloWorldFunction at http://127.0.0.1:3000/id [POST]
2018-05-09 18:42:33 Mounting EchoEventFunction at http://127.0.0.1:3000/id/{id}/user/{user} [GET]
2018-05-09 18:42:33 Mounting SleepFunction1 at http://127.0.0.1:3000/sleepfortenseconds/function1 [GET]
2018-05-09 18:42:33 Mounting EchoEventFunction at http://127.0.0.1:3000/id/{id} [GET]
2018-05-09 18:42:33 Mounting ContentTypeSetterFunction at http://127.0.0.1:3000/getcontenttype [GET]
2018-05-09 18:42:33 Mounting SleepFunction0 at http://127.0.0.1:3000/sleepfortenseconds/function0 [GET]
2018-05-09 18:42:33 Mounting WriteToStdoutFunction at http://127.0.0.1:3000/writetostdout [GET]
2018-05-09 18:42:33 Mounting InvalidResponseFromLambdaFunction at http://127.0.0.1:3000/invalidresponsereturned [GET]
```

This issue was addressed in the [0.3.0 release](https://github.com/awslabs/aws-sam-cli/releases/tag/v0.3.0). 

Closing
